app/relay-route/CityBikeRoute.js

```diff
@@ -0,0 +1,17 @@
+import Relay from 'react-relay';
+
+export default class CityBikeRoute extends Relay.Route {
+  /* eslint-disable graphql/template-strings */
```

Comment:
Why are you disabling the rule. Shouldn't you just configure the variable for the query to fix the issue?